packages/host/app/components/matrix/room-message.gts

```diff
@@ -308,6 +310,24 @@ export default class RoomMessage extends Component<Signature> {
       .join(', ');
   }
 
+  //TODO: Will remove this
```

Comment:
Is this to be removed in this PR or in the target feature branch?

Reply:
When search is implemented and the component have a generalised notion of command. We can check using `isCommand` rather than `isPatchCommand`. This PR, doesn't have apply button for search working (which is a command) so I didn't want to expose the Apply button for search